zebra-network/src/constants.rs

```diff
@@ -95,6 +95,15 @@ pub const EWMA_DEFAULT_RTT: Duration = Duration::from_secs(20 + 1);
 /// better peers when we restart the sync.
 pub const EWMA_DECAY_TIME: Duration = Duration::from_secs(200);
 
+/// The maximum time to open a port if it haves a timeout.
```

Comment:
```suggestion
/// The maximum time to open a listener port.
///
/// Opening listener ports can block on some OSes. The default timeout can be very long.
```

Reply:
I will probably add this at https://github.com/ZcashFoundation/zebra/issues/1631 as i removed the constant from this PR.